docker-example/mc-docker.yaml

```diff
@@ -0,0 +1,80 @@
+tosca_definitions_version: cloudify_dsl_1_3
+
+description: Cloudify Docker blueprint. Deploys Docker.
+
+imports:
+  - http://cloudify.co/spec/cloudify/5.0.5/types.yaml
+  - plugin:cloudify-fabric-plugin
+
+inputs:
+  infra_name:
+    description: >
+      Name of infrastructure blueprint to deploy.
+    type: string
+    constraints:
+      - valid_values:
+          - openstack
+          - azure
+          - azure-arm
+          - aws
+          - aws-terraform
+          - aws-cloudformation
+          - gcp
+
+  infra_archive:
+    description: >
+      URL of infra zip file.
+    type: string
+    # Don't update this line - it is being updated automatically during merging. The getting-started.zip will have the updated URL.
+    default: 'https://github.com/cloudify-community/blueprint-examples/releases/download/5.0.5-10/virtual-machine.zip'
```

Comment:
@EarthmanT need to add that to the release management and see that this link is being updated on this example ?

Reply:
It should already be in the release management.